server/dash/dashboards/neighborhood.py

```diff
@@ -39,27 +42,80 @@ def populate_options():
         )
     return values
 
+print(populate_options())
+
+layout = html.Div([
+    html.H1(title),
+    dcc.Dropdown(
+        id='council_list',
+        clearable=False,
+        value="Arleta",
+        placeholder="Select a neighborhood",
+        options=populate_options()
+    ),
+    dcc.Dropdown(
+        id='data_type',
+        clearable=False,
+        placeholder="Select a timeframe",
+        options=[
+            {'label': 'Compare to: Daily Council Average', 'value': 'nc_avg'},
+            {'label': 'Compare to: Weekly Council Average', 'value': 'nc_avg_wk'},
+            {'label': 'Compare to: Monthly Council Average', 'value': 'nc_avg_Mo'},
+        ],
+
+        value='nc_avg'
+    ),
+    dcc.Graph(
+        id='graph1',
+        figure=fig,
+        config=CONFIG_OPTIONS
+    ),
+
+    dcc.Graph(
+        id='graph2',
+        figure=fig,
+        config=CONFIG_OPTIONS
+    )
+])
+
 
 # Define callback to update graph
 @app.callback(
     Output('graph1', 'figure'),
-    [Input("council_list", "value")]
+    [Input("council_list", "value"),
+    Input("data_type", "value")]
 )
-def update_figure(selected_council):
-
+def update_figure(selected_council, select_timeframe):
     neighborhood_sum_df = df[df.council_name == selected_council].groupby(['created_date']).agg('sum').reset_index()  # noqa
+
     total_sum_df = df.groupby(['created_date']).agg('sum').reset_index()
+
     total_sum_df["nc_avg"] = total_sum_df["counts"] / 99
-    merged_df = neighborhood_sum_df.merge(total_sum_df["nc_avg"].to_frame(), left_index=True, right_index=True)  # noqa
 
+    total_sum_df["week_n"] = total_sum_df['created_date'].apply(lambda x: str(x.isocalendar()[1]))
```

Comment:
Could you leave some comments explaining what week_n and month_n are?

Reply:
These are no longer needed.